docs/docs/canonicai/chapter6.md

```diff
@@ -5,87 +5,120 @@
 ## Bringing your application to production
 
 ### Register and update your jac code on a remote instance
+
 In this section, we will walk you through on how to register and update your jac code on a remote instance. The steps are as follows:
 
+### Managing your application with Jaseci Studio
```

Comment:
@Gim3l I don't think this works here. Because at this point, we have not talked about jsserv or using it to create a user. That comes later in this doc.  Could you move this to later in the doc where the jsserv is introduced?